src/features/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  value: '',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state) => {
+      state.value = state.value === '' ? 'Under construction' : null;
+    },
+  },
+});
```

Comment:
- [ ] Great job here, but I noticed a couple of things that could be improved. Firstly, for the initial state of the categories, it should be an empty array instead of a string, since you will be storing multiple categories depending on the book. Please make sure to update this for consistency and to avoid potential issues down the line.
- [ ] Additionally, in the `checkStatus` reducer at line `11` - `12`, you are updating the state value using the state value itself, but instead, you should accept an action as a parameter and check the payload to update the state value. This will ensure that your reducer behaves correctly and follows the principles of Redux. Keep up the good work!